src/EPR.Payment.Facade/Controllers/Payments/OnlinePaymentsController.cs

```diff
@@ -14,19 +14,19 @@ namespace EPR.Payment.Facade.Controllers.Payments
 {
     [ApiVersion(1)]
     [ApiController]
-    [Route("api/v{version:apiVersion}/payments")]
+    [Route("api/v{version:apiVersion}/onlinepayments")]
```

Comment:
Same, should be online-payments

Reply:
it is  [Route("api/v{version:apiVersion}/online-payments")] now